src/_utils/crm-web-api.ts

```diff
@@ -345,13 +360,20 @@ export const CRMWebAPI = {
       })
   },
 
+  /** 
+    * Finds folder for an entity (like `dcp_project` or `dcp_communityboarddisposition`).
+    * Entity folders contain many instance folders, the paths of which can be 
+    * acquired instead via findDocumentLocation.
+    * @param { GUID } entityName - name of CRM entity, like  `dcp_project` or `dcp_communityboarddisposition`.
+    * @param { string } sharepointSiteID - ID of CRM's corresponding sharepoint site. Use getParentSiteLocation() to acquire this ID
+  */
```

Comment:
Bonus points would be to convert this documentation into TypeScript definitions 😇